mungegithub/mungers/submit-queue.go

```diff
@@ -230,6 +235,11 @@ func (sq *SubmitQueue) e2eStable() bool {
 	wentUnstable := false
 
 	stable := sq.e2e.Stable()
+	gcsStable := sq.e2e.GCSBasedStable()
+
+	if !gcsStable {
+		glog.Errorf("Weak stable check returned error.")
```

Comment:
Is this an error? seems like stable != gcsStable is an error. But gcsStable == false seem 'normal'....